doc/examples_sphinx-gallery/stochastic_variability.py

```diff
@@ -0,0 +1,93 @@
+"""
+.. _tutorials-stochastic-variability:
+
+=========================================================
+Stochastic Variability in Community Detection Algorithms
+=========================================================
+
+This example demonstrates the variability of stochastic community detection methods by analyzing the consistency of multiple partitions using similarity measures (NMI, VI, RI) on both random and structured graphs.
+
+"""
+# %%
+# Import Libraries
+import igraph as ig
+import numpy as np
+import matplotlib.pyplot as plt
+import itertools
+
+# %%
+# First, we generate a graph.
+# Generates a random Erdos-Renyi graph (no clear community structure)
+def generate_random_graph(n, p):
+    return ig.Graph.Erdos_Renyi(n=n, p=p)
```

Comment:
Do not omit diacritics. It is Erdős-Rényi.

For clarity, do indicate that it is an Erdős-Rényi $G(n,p)$ graph (i.e. not $G(n,m)$).

Do we really need to define new functions to generate these graphs? This function just wraps `Graph.Erdos_Renyi`.